packages/@aws-cdk/aws-ivs/README.md

```diff
@@ -9,8 +9,47 @@
 ---
 <!--END STABILITY BANNER-->
 
+Amazon Interactive Video Service (Amazon IVS) is a managed live streaming solution that is quick and easy to set up, and ideal for creating interactive video experiences. Send your live streams to Amazon IVS using streaming software and the service does everything you need to make low-latency live video available to any viewer around the world, letting you focus on building interactive experiences alongside the live video. You can easily customize and enhance the audience experience through the Amazon IVS player SDK and timed metadata APIs, allowing you to build a more valuable relationship with your viewers on your own websites and applications.
+
 This module is part of the [AWS Cloud Development Kit](https://github.com/aws/aws-cdk) project.
 
+### Installation
+
+Import to your project:
+
 ```ts
 import ivs = require('@aws-cdk/aws-ivs');
 ```
+
+### Basic usage
+
+You can create a channel
```

Comment:
introduce the concept of channels, what they represent and then lead into it's creation code.